### PR TITLE
Setup environment for packaging

### PR DESCRIPTION
**Pull request by sodrooome**

### CODE REVIEW

**Comment by sodrooome** [comment]
Can you please review it? @andigortz

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import setuptools
+
+import io
+import os
+import re
+
+with io.open('lru/version.py', 'rt', encoding='utf-8') as f:
+    version = re.search(r"__version__ = '(.*?)'", f.read()).group(1)
```
Comment (andigortz) [inline setup.py]: i think you got an mistaken in here

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    name="lruheap",
```
Comment (andigortz) [inline setup.py]: and this too

**Review by andigortz — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (sodrooome) [reply setup.py]: oh, my bad. im gonna fix this later

**Comment by sodrooome** [comment]
All fixed with ecf63e5

**Pull request merged**